pkgs/by-name/pr/prismlauncher-unwrapped/package.nix

```diff
@@ -4,12 +4,11 @@
 , stripJavaArchivesHook
 , cmake
 , cmark
-, Cocoa
+, darwin
 , ninja
 , jdk17
 , zlib
-, qtbase
-, quazip
+, kdePackages
```

Comment:
Can this be changed to qt5 or qt6?

Reply:
it can be, but shouldn't. we're currently in the process of deprecating distribution of builds against qt 5 in distros where possible (and reasonable ofc) upstream, up until it's eventual removal

if users *really* want to use qt 5 still for whatever reason, i think using the standard override interface here would suffice

Reply:
So, can this then be changed to qt6?

As I have seen at the website, this application does not use KDE-specific things, only a small bunch of Qt libs...

P.S.: can you point me to a specific place where the Qt5 deprecation is discussed? I have some Qt6+5 packages to deal with...

Reply:
> So, can this then be changed to qt6?

is `kdePackages` not qt 6? i'm a bit confused here

> P.S.: can you point me to a specific place where the Qt5 deprecation is discussed? I have some Qt6+5 packages to deal with...

we're tracking this upstream at https://github.com/PrismLauncher/PrismLauncher/issues/2324 with the actual removal in https://github.com/PrismLauncher/PrismLauncher/pull/2174. we dropped support for our qt 5 build here in https://github.com/NixOS/nixpkgs/pull/305556